pkg/dependenciesdistributor/dependencies_distributor.go

```diff
@@ -336,9 +338,11 @@ func (d *DependenciesDistributor) ReconcileResourceBinding(key util.QueueKey) er
 
 	dependencies, err := d.ResourceInterpreter.GetDependencies(workload)
 	if err != nil {
+		d.EventRecorder.Eventf(workload, corev1.EventTypeWarning, workv1alpha2.EventReasonGetDependenciesFailed, "Get dependencies failed, err: %s.")
```

Comment:
A small question: format selection for failed event information：
1. xxxx failed, err : %s
2. err.Error()

I'm not sure which format is better.